unpub/example/main.dart

```diff
@@ -14,6 +14,6 @@ main(List<String> args) async {
     packageStore: packageStore,
   );
 
-  var server = await app.serve('0.0.0.0', 4000);
+  var server = await app.serve('0.0.0.0', 8000);
```

Comment:
Are these changes related to the style? If not, perhaps it's better to put them in another PR.

Reply:
these changes is not necessarily for the style.

Reply:
@pd4d10 I've removed unnecessary changes